src/courtroom.cpp

```diff
@@ -1833,6 +1833,13 @@ void Courtroom::list_areas()
         else if (arup_statuses.at(n_area) == "GAMING")
         {
           treeItem->setBackground(1, gaming_brush);
+        }        
+        else if (arup_statuses.at(n_area) != "") {
+          QString status = arup_statuses.at(n_area).toLower();
+          QString custom_status_color = "area_" + status + "_color";
+          custom_status_color = ao_app->check_status_color(custom_status_color, "courtroom_design.ini");
+          custom_status_brush = QBrush(ao_app->get_color(custom_status_color, "courtroom_design.ini"));
```

Comment:
I cannot set there custom_status_color because there could be more than 1 custom color.
So I cannot write it like "free_brush = QBrush(ao_app->get_color("area_free_color", "courtroom_design.ini"))"
It is necessary that variable changes everytime during constructing area list it sees a new status

Reply:
I assumed that `custom_status_brush` would be used if a custom status is found but not defined, as a fallback.

You definitely can write it like I mentioned if you write the methods like I said below, e.g. `defaultColor` etc.